### PR TITLE
[JS]Cleaning up chrome.js file

### PR DESCRIPTION
**Pull request by harsha509**
### Description
Removing duplicate declarations from chrome as chrome extends chromium.js

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: javascript/node/selenium-webdriver/chrome.js [hunk 1]
```diff
@@ -346,7 +209,7 @@ class Options extends chromium.Options {
   }
 
   /**
-   * Sets the directory to store Chrome minidumps in. This option is only
+   * Sets the directory to store Chrome minidump in. This option is only
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/chrome.js]: `minidumps` is the correct term

**Review by AutomatedTester — APPROVED** [review]
Could we have something like the description as the commit message so that we can see what the changes are when we look at the git log.

File: javascript/node/selenium-webdriver/chrome.js (hunk 1 above)
Comment (harsha509) [reply javascript/node/selenium-webdriver/chrome.js]: Thank you for reviewing @AutomatedTester. I have reverted it.

**Comment by harsha509** [comment]
> Could we have something like the description as the commit message so that we can see what the changes are when we look at the git log.

Thank you for the review @AutomatedTester. I have updated the git commit message.

**Review by diemol — APPROVED** [review]

**Pull request merged**